node/impl/full/eth_utils.go

```diff
@@ -88,7 +88,7 @@ func getTipsetByBlockNumber(ctx context.Context, chain *store.ChainStore, blkPar
 			return nil, fmt.Errorf("cannot get tipset at height: %v", num)
 		}
 		if strict && ts.Height() != abi.ChainEpoch(num) {
-			return nil, ErrNullRound
+			return nil, api.NewErrNullRound(int64(num))
```

Comment:
how about you just make it take an `abi.ChainEpoch` instead since that's what it would always be anyway